torch/csrc/utils/tensor_flatten.cpp

```diff
@@ -67,31 +116,37 @@ at::Tensor get_values(const at::Tensor& t) {
   return t._values();
 }
 
-}
+} // namespace
 
-std::pair<at::Tensor, at::Tensor> flatten_sparse_tensors(at::TensorList tensors) {
+std::pair<at::Tensor, at::Tensor> flatten_sparse_tensors(
+    at::TensorList tensors) {
   auto flat_indices = flatten_dense_tensors(fmap(tensors, &get_indices));
   auto flat_values = flatten_dense_tensors(fmap(tensors, &get_values));
   return std::make_pair(flat_indices, flat_values);
 }
 
 std::vector<at::Tensor> unflatten_sparse_tensors(
-        const at::Tensor& flat_indices, const at::Tensor& flat_values,
-        at::TensorList tensors) {
-  if (tensors.size() == 0) return {};
+    const at::Tensor& flat_indices,
+    const at::Tensor& flat_values,
+    at::TensorList tensors) {
+  if (tensors.size() == 0)
+    return {};
 
-  auto indices = unflatten_dense_tensors(flat_indices, fmap(tensors, &get_indices));
-  auto values = unflatten_dense_tensors(flat_values, fmap(tensors, &get_values));
+  auto indices =
+      unflatten_dense_tensors(flat_indices, fmap(tensors, &get_indices));
+  auto values =
+      unflatten_dense_tensors(flat_values, fmap(tensors, &get_values));
```

Comment:
Please avoid reformatting code you’re not touching... it’s not a good style, makes your diff unnecessarily larger, and makes reviewing harder because it adds noise